hosts.ose

```diff
@@ -0,0 +1,725 @@
+# This is an example of a bring your own (byo) host inventory
```

Comment:
slightly confused why this file is at the project root instead of under inventory/byo.
It might make sense to either remove this file or add to one of the current host examples.